### PR TITLE
Updating topics page

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

https://trello.com/c/o3MMYeGZ

### Intent

Move topics page over to just show topics rather than tags. 
Do not display description:

Screenshot 2022-03-16 at 14 16 08
 

### Considerations

 Will need to follow up to change content types to topics.

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Review by jonhindle — APPROVED** [review]
lgtm

**Review by jonhindle — DISMISSED** [review]
lgtm

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Review by jonhindle — APPROVED** [review]
lgtm

File: server/middleware/retrieveTopicList.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    if (!establishmentName) {
+      throw new Error('Could not determine establishment!');
+    }
+    res.locals.allTopics = await cmsService.getTopics(establishmentName);
+    next();
+  } catch (e) {
+    e.message = `Error loading topics: ${e.message}`;
+    next(e);
+  }
+};
```
Comment (andrewrlee) [inline server/middleware/retrieveTopicList.js:14]: @jonhindle  we've altered some tests to cover it but not testing it anger - we'll need to think if we can both use the same middleware and merge this into yours or not.

File: server/routes/index.js [hunk 2]
```diff
@@ -30,6 +31,23 @@ module.exports = (
...
+      '/tags',
+      '/topics',
+      '/timetable',
+      '/money',
+      '/approved-visitors',
+      '/profile',
+      '/games',
+      '^/search/?$',
+    ],
+    retrieveTopicList(cmsService),
```
Comment (andrewrlee) [inline server/routes/index.js:48]: @jonhindle  you're list vs ours: 

```
'/', '/topics', '/timetable', '/money', '/approved-visitors', '/profile', '/content', '/npr', '/tags', '/games']
```

think its just search that differs? we're ensuring it doesn't fire for the suggestion api, maybe that would suffice for you?

**Review by jonhindle — APPROVED** [review]
lgtm

**Pull request merged**